### ISSUE TITLE
[DOCUMENT]: GIBCT SCO - Dear SCO letter list SHOULD announce as a list

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#8764]
**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

VA 508 office has identified the [Dear School Certifying Official](https://www.benefits.va.gov/GIBILL/docs/fgib/107_TandF_SCO_letter.pdf) PDF has a list that is not announcing as a bulleted list to screen readers. Screen shot attached below.

## Related Issue
* #3918
* #8758 
* #8761 

## Point of Contact

**VFS Point of Contact:** _Trevor_

## Acceptance Criteria

* [ ] Acrobat Pro accessibility checker validates a tagged list
* [ ] List is announcing properly to screen readers

## Screenshots or Trace Logs
![Screen Shot 2020-05-05 at 11 54 18 AM](https://user-images.githubusercontent.com/934879/81093142-47a73d00-8ec7-11ea-914a-e30012379b5c.png)

**Comment by desireeturner** [comment]
@TamGIBill I believe this is content that you can updated. Please let me know if it is not. Thank you!

**Comment by TamGIBill** [comment]
Letter has been removed.
